pvdeg/standards.py

```diff
@@ -139,36 +256,178 @@ def standoff(
+    x_0 : float, optional
+        Thermal decay constant. [cm]
+    weather_df : pd.DataFrame
+        Weather data for a single location. 
+    meta : pd.DataFrame
+        Meta data for a single location.
+    tilt : float, 
+        Tilt angle of PV system relative to horizontal. [°]
+    azimuth : float, optional
+        Azimuth angle of PV system relative to north. [°]
+    sky_model : str, optional
+        Options: 'isotropic', 'klucher', 'haydavies', 'reindl', 'king', 'perez'.
+    temp_model : str, optional
+        Options: 'sapm'.  'pvsyst' and 'faiman' will be added later.
+        Performs the calculations for the cell temperature.
+    conf_0 : str, optional
+        Model for the high temperature module on the exponential decay curve.
+        Default: 'insulated_back_glass_polymer'
+    conf_inf : str, optional
+        Model for the lowest temperature module on the exponential decay curve.
+        Default: 'open_rack_glass_polymer'
+    wind_factor : float, optional
+        Wind speed correction exponent to account for different wind speed measurement heights
+        between weather database (e.g. NSRDB) and the tempeature model (e.g. SAPM)
+        The NSRDB provides calculations at 2 m (i.e module height) but SAPM uses a 10 m height.
+        It is recommended that a power-law relationship between height and wind speed of 0.33 
+        be used. This results in a wind speed that is 1.7 times higher. It is acknowledged that 
+        this can vary significantly. 
+    
+    Returns
+    -------
+    T98: float
+        This is the 98th percential temperature for the module at the given tilt, azimuth, and x_eff.
+
+    """
+    if azimuth == None: #Sets the default orientation to equator facing.
+        if float(meta['latitude']) < 0:
+            azimuth=0
+        else:
+            azimuth=180
+    if 'Wind_Height_m' not in meta.keys():
+        wind_factor = 1
+    parameters = ["temp_air", "wind_speed", "dhi", "ghi", "dni"]
+
+    if isinstance(weather_df, dd.DataFrame):
+        weather_df = weather_df[parameters].compute()
+        weather_df.set_index("time", inplace=True)
+    elif isinstance(weather_df, pd.DataFrame):
+        weather_df = weather_df[parameters]
+    elif weather_df is None:
+        weather_df, meta = weather.get(**weather_kwarg)
+
+    solar_position = spectral.solar_position(weather_df, meta)
+    poa = spectral.poa_irradiance(
+        weather_df=weather_df,
+        meta=meta,
+        sol_position=solar_position,
+        tilt=tilt,
+        azimuth=azimuth,
+        sky_model=sky_model, )
+    T_0 = temperature.cell(
+        weather_df, meta, poa, temp_model, conf_0, wind_factor, )
+    T98_0 = T_0.quantile(q=0.98, interpolation="linear")
+    T_inf = temperature.cell(
+        weather_df, meta, poa, temp_model, conf_inf, wind_factor, )
+    T98_inf = T_inf.quantile(q=0.98, interpolation="linear")
+
+    T98 = T98_0 - (T98_0-T98_inf)*(1-np.exp(-x_eff/x_0))
```

Comment:
x_eff is always None in the current implementation

Reply:
I fixed it so that it would default to an open rack if x_eff is not specified. While I was at it, I also set the default for the tilt not being specified to use the latitude tilt.

Reply:
thanks. yes, that's looking good to me